### DISCUSSION TITLE
Communicate using markdown series problem

### DISCUSSION THREAD

**Discussion by uw935** [community/community#108207] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey there!

In this course I have 6 step:
"Select Edit file from the three dotted ... menu in the upper right corner of the file view on index.md."

But when I try to check that, Edit file button is grey, what should I do?

![image](https://github.com/community/community/assets/74175088/293f5470-89ab-4b0f-b56a-f77d578b0f99)

**Comment by jmarintur** [comment]
Hi @uw935, did you sign in using the same account you used to create the index.md? In this repo, what permissions do you have?

**Reply by uw935** [reply]
Yes, I logged in with the same account and I have the full permissions, I even can delete this repo

**Reply by jmarintur** [reply]
If the file is being viewed as part of a specific tag or a past commit, GitHub does not provide an edit option directly in the UI for these instances.

**Reply by jmarintur** [reply]
Here I am showing you a past file I changed in a PR, that I cannot edit anymore. 

![image](https://github.com/community/community/assets/28574023/607f4f6b-840e-4a28-ba83-222467293596)

**Reply by uw935** [reply]
I don't know, I haven't done anything with this file.

This repository: https://github.com/Exp-Communicate-Using-Markdown-Cohort-1/series-communicate-using-markdown-uw935

**Reply by jmarintur** [reply, marked as answer]
It seems to me that you are viewing the file from a past commit (the screenshot you provided didn't show the whole thing). Can you go to the code, and open the file directly? 

![image](https://github.com/community/community/assets/28574023/142b43e0-f287-47f0-87fe-78f21bab24bc)

**Reply by uw935** [reply]
Oh, yes, it works from the directly opening this file in the code. Thank you!
